js/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "omi-summer-lab",
+  "version": "1.0.0",
+  "description": "none",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/IntelLedger/omi-summer-lab.git"
+  },
+  "main": "lib/index.js",
+  "scripts": {
+    "compile-protobuf": "node ./scripts/compile_protobuf.js > ./lib/protobuf/protobuf_bundle.json",
+    "test": "standard && mocha --recursive spec --compilers js:babel-register -r babel-polyfill",
+    "watch:test": "mocha -w --recursive spec --compilers js:babel-register -r babel-polyfill",
+    "postinstall": "./scripts/install_sawtooth_sdk.sh && npm run compile-protobuf",
```

Comment:
Keep in mind, this is also a bit of a hacky work-around until we complete [STL-281](https://jira.hyperledger.org/browse/STL-281): Publish JS SDK to NPM registry